src/main/java/me/botsko/prism/database/mysql/MySqlPrismDataSource.java

```diff
@@ -106,6 +105,11 @@ private static void setupDefaultProperties(@Nonnull ConfigurationSection section
                 }
             }
             try {
+                if (!propFile.getParentFile().exists()) {
+                    if (!propFile.getParentFile().mkdirs()) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5472183)